docs/CaliptraIntegrationSpecification.md

```diff
@@ -195,6 +195,7 @@ Although fuse values (and the fuse done register) persist across a warm reset, S
 ## Interface rules
 
 The following figure shows the reset rules and timing.
+The steps illustrated should be followed by the SoC for cold boot flows.
```

Comment:
There is no explicit mention of the use of brkpoint/bootfsm_go in this diagram, or anywhere else in the spec really. I recommend one of these options
1) Updating the diagram to indicate these (optional) steps 
2) Updating the bootfsm diagram to include the breakpoint operation
3) Adding text description under this heading or under the [bootfsm](https://github.com/chipsalliance/caliptra-rtl/blob/631fa475289a0b416c91c1133bf83f5771687b48/docs/CaliptraIntegrationSpecification.md#boot-fsm) heading that explains breakpoint operation. Maybe in both places.

Reply:
I added some verbage to the boot flow regarding this.

I'll try to update the bootfsm diagram this week also.

Reply:
Thanks! I recommend changing the language from future tense to present tense to match the rest of the doc, but otherwise looks good.